### ISSUE TITLE
Publish for 2.13

### ISSUE THREAD

**Issue by gabro** [scalameta/scalameta#1873]
2.13.0-RC1 support is already done (https://github.com/scalameta/scalameta/pull/1870) so it should be a matter of testing that it still works and publishing.

Currently blocked by scalapb (which has merged 2.13 on master a couple of hours ago, but has not published artifacts yet)
